Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -227,3 +227,23 @@ public enum AuthMechanism: String, Decodable {
     case mongodbX509 = "MONGODB-X509"
     case plain = "PLAIN"
 }
+
+public let INVALID_ARGUMENT_ERROR = InvalidArgumentError(message: "")
```

Comment:
these error cases don't compare the messages when checking for equality, so I just made some singletons that can be used in the tests.